src/Model/Index/SearchIndex.php

```diff
@@ -6,6 +6,8 @@
 
 class SearchIndex extends Index
 {
+    protected const MAX_LIMIT = 25;
```

Comment:
Clamping max limit to default limit seems quite limiting 🙂.

Reply:
I thought about not exceeding the limit that you already had before

so.. how do you want it?

Reply:
Meh.. 25 is fine. We can see later if a higher value is actually necessary.